api/models.py

```diff
@@ -42,22 +54,29 @@ class Item(models.Model):
         ('project', 'project'),
         ('preprint', 'preprint'),
         ('registration', 'registration'),
-        ('meeting', 'meeting'),
-        ('website', 'website')
+        ('presentation', 'presentation'),
+        ('website', 'website'),
+        ('event', 'event')
     )
     STATUS = (
         ('approved', 'approved'),
         ('pending', 'pending'),
         ('rejected', 'rejected')
     )
-    source_id = models.TextField()
     title = models.TextField()
+    description = models.TextField(null=True, blank=True)
     type = models.TextField(choices=TYPES)
-    status = models.TextField(choices=STATUS)
-    url = models.URLField()
-    collection = models.ForeignKey(to='Collection', related_name='items')
+    status = models.TextField(choices=STATUS, null=True)
+    source_id = models.TextField(null=True, blank=True)
+    url = models.URLField(null=True, blank=True)
+    collection = models.ForeignKey(to='CollectionBase', related_name='items')
     group = models.ForeignKey(to='Group', null=True, blank=True, default=None, related_name='items')
     created_by = models.ForeignKey(User)
-    metadata = JSONField()
-    date_added = models.DateTimeField(null=True, blank=True, default=None)
-    date_submitted = models.DateTimeField(auto_now_add=True)
+    metadata = JSONField(null=True, blank=True)
+    date_created = models.DateTimeField(auto_now_add=True)
+    date_submitted = models.DateTimeField(null=True, blank=True, default=None)
+    date_accepted = models.DateTimeField(null=True, blank=True, default=None)
+    location = models.TextField(null=True, blank=True, default=None)
+    start_time = models.DateTimeField(null=True, blank=True, default=None)
+    end_time = models.DateTimeField(null=True, blank=True, default=None)
+    category = models.TextField(null=True, blank=True, default=None)
```

Comment:
That sounds good to me! And we can add more if needed. As mentioned above, might as well change it to a CharField too then :)